WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -89,16 +90,28 @@ class PostListItemUiStateHelper @Inject constructor(
         val onButtonClicked = { buttonType: PostListButtonType ->
             onAction.invoke(post, buttonType, POST_LIST_BUTTON_PRESSED)
         }
-        val buttonTypes = createButtonTypes(
+        val defaultButtonTypes = createButtonTypes(
```

Comment:
I feel like it might be a bit difficult to read when we call the same method (which unfortunately needs a lot of parameters) twice in a row and we change just one of the parameters. How would you feel about something like this

```
val getButtonTypes: (isCompact: Boolean) -> List<PostListButtonType> = { isCompact ->
            createButtonTypes(
                    postStatus = postStatus,
                    isLocalDraft = post.isLocalDraft,
                    isLocallyChanged = post.isLocallyChanged,
                    uploadUiState = uploadUiState,
                    siteHasCapabilitiesToPublish = capabilitiesToPublish,
                    statsSupported = statsSupported,
                    isCompactActions = isCompact
            )
        }
        val defaultActions = createDefaultViewActions(getButtonTypes(false), onButtonClicked)
        val compactActions = createCompactViewActions(getButtonTypes(true), onButtonClicked)
```